cabot/cabotapp/models.py

```diff
@@ -9,13 +9,14 @@
 from .influx import parse_metric
 from .tasks import update_service
 from cabot.cabotapp.models_plugins import HipchatInstance
+from cabot.cabotapp.defs import *
```

Comment:
nit: `import *` is usually [frowned upon](https://shahriar.svbtle.com/importing-star-in-python), but we could do
`from cabot.cabotapp import defs`
and then refer to 
`defs.DEFAULT_HTTP_TIMEOUT`, etc.